client/scss/base/_layout.scss

```diff
@@ -7,13 +7,7 @@
 }
 
 .is-mobile-nav-visible {
-  position: fixed;
-  width: 100%;
-  height: 100%;
-
-  @include respond-to('medium') {
-    position: inherit;
-    width: inherit;
-    height: inherit;
+  [role='main'] {
```

Comment:
We could create a class. I thought it was quite descriptive to use the pre-existing `role` though.